### ISSUE TITLE
PackageNotFoundError in Manjaro after last pacman update

### ISSUE THREAD

**Issue by mroavi** [ssokolow/quicktile#120]
I just updated my packages using `sudo pacman -Syu` in Manjaro. As part of this update, Python was upgraded to 3.9.1. I suspect this is the reason for the following error when starting quicktile:

```zsh
❯ quicktile --daemonize
Traceback (most recent call last):
  File "/usr/bin/quicktile", line 33, in <module>
    sys.exit(load_entry_point('QuickTile==0.4.1', 'console_scripts', 'quicktile')())
  File "/usr/bin/quicktile", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/lib/python3.9/importlib/metadata.py", line 524, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/lib/python3.9/importlib/metadata.py", line 187, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: QuickTile
```

**Comment by ssokolow** [comment]
That's the classic `setuptools` fragility that prompted me to write the [I get an error when I try to run QuickTile](http://ssokolow.com/quicktile/faq.html#i-get-an-error-when-i-try-to-run-quicktile) FAQ entry.

**TL;DR:** It's a design flaw in how Python's `setup.py` system works, not a QuickTile bug. Completely uninstall QuickTile and reinstall it.

Did you already try those instructions?

**Comment by mroavi** [comment]
I reinstalled quicktile with
```zsh
sudo pip3 install https://github.com/ssokolow/quicktile/archive/master.zip
```
 and now it is working. Thanks.
